lambdas/service/app.py

```diff
@@ -1097,8 +1097,8 @@ def dss_files(uuid):
                 here is intended to be processed by client-side Javascript such
                 that the recommended delay in `Retry-After` can be handled in
                 Javascript rather that relying on the native implementation by
-                the web browser. See `/dss/files` for the meaning of the
-                response elements.
+                the web browser. See [/dss/files](#/DSS/get_dss_files__uuid_)
+                for the meaning of the response elements.
```

Comment:
```suggestion
                the web browser.

                For a detailed description of the fields in the response see
                the documentation for the headers they emulate in the
                [`/dss/files`](#operations-DSS-get_dss_files__uuid_)
                endpoint response.                
```
This anchor link will jump to the endpoint without a new tab, tested in Chrome & FireFox. The link target is in reference to the `id` field specified in the outer div of the endpoint block:
```
<div class="opblock opblock-get is-open" id="operations-DSS-get_dss_files__uuid_">
```


Also, the first paragraph is a bit much for the reader to take in, so I recommend putting the link in a second paragraph with some verbiage to reiterate the relationship between the fields of this response to the headers of the other endpoint's response.

Reply:
Awesome, thanks. I did try looking at the source but somehow missed that ID field.